frontends/web/src/components/forms/select.tsx

```diff
@@ -29,14 +29,12 @@ type TSelectProps = {
     id: string;
     label?: string;
     options: TOption[];
-    selectedOption?: string;
 } & JSX.IntrinsicElements['select']
```

Comment:
Looks good, if I am not mistaken you can also remove `defaultValue` in type TSelectProps. This was added because Preact 8 didn't type it properly (or didn't have support).

But in React defaultValue is supported on select element https://reactjs.org/docs/uncontrolled-components.html#default-values and it will be passed through `{...props}`.

```diff
diff --git a/frontends/web/src/components/forms/select.tsx b/frontends/web/src/components/forms/select.tsx
index 7055c6798..685f277a2 100644
--- a/frontends/web/src/components/forms/select.tsx
+++ b/frontends/web/src/components/forms/select.tsx
@@ -24,8 +24,6 @@ type TOptionTextContent = {
 type TOption = JSX.IntrinsicElements['option'] & TOptionTextContent
 
 type TSelectProps = {
-    // Temp add defaultValue, see https://github.com/preactjs/preact/issues/2668
-    defaultValue?: string;
     id: string;
     label?: string;
     options: TOption[];
```